frontend/src/model/Tile.ts

```diff
@@ -0,0 +1,20 @@
+import Locations from '../constants/locations';
+
+class Tile {
+  edgeLeft: Locations;
+
+  edgeRight: Locations;
+
+  edgeTop: Locations;
+
+  edgeBottom: Locations;
+
+  constructor(edgeLeft: Locations, edgeRight: Locations, edgeTop: Locations, edgeBottom: Locations) {
```

Comment:
This list will be growing fast - let's replace it with a single object called (it may be called edges)